quantized_mesh_encoder/extensions.py

```diff
@@ -41,7 +49,55 @@ def encode(self) -> bytes:
         encoded = oct_encode(normals).tobytes('C')
 
         buf = b''
-        buf += pack(EXTENSION_HEADER['extensionId'], ExtensionId.VERTEX_NORMALS)
+        buf += pack(EXTENSION_HEADER['extensionId'], self.id.value)
+        buf += pack(EXTENSION_HEADER['extensionLength'], len(encoded))
+        buf += encoded
+
+        return buf
+
+
+@attr.s(kw_only=True)
+class WaterMaskExtension(ExtensionBase):
+    id: ExtensionId = attr.ib(
+        ExtensionId.WATER_MASK,
+        validator=attr.validators.instance_of(ExtensionId))
+    data: Union[np.ndarray, np.uint8, int] = attr.ib(
+        validator=attr.validators.instance_of((np.ndarray, np.uint8, int)))
```

Comment:
Might want to validate that if the input is an `np.ndarray`, then the shape is `(256, 256)`.